### DISCUSSION TITLE
[Esame 31 Marzo 2023] Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#356] [Esami - Bandiera]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/4f517b30-28b3-4b45-8d53-b22cf1f43c73)

**Comment by interestingmaneuver** [comment]
$U_9 = \\{ n \in \mathbb{Z} | 1 \le n \lt 9, (n, 9) = 1 \\} = \\{ 1, 2, 4, 5, 7, 8 \\}$
$U_{14} = \\{ 1, 3, 5, 9, 11, 13 \\}$

$U_9$ e $U_{14}$ sono ciclici, infatti $U_9$ è generato da $2$ ($2^6 \equiv 1 \mod 9$), e $U_{14}$ è generato da $3$ ($3^6 \equiv 1 \mod 14$).

Per trovare i generatori si può andare in ordine, siccome per il teorema di Lagrange un gruppo di ordine 6 può avere elementi di ordine 1, 2, 3, o 6, appena si vede che l'ordine è $> 3$ si può concludere che l'elemento genera il gruppo.

Sia $G$ è un gruppo ciclico finito, è possibile definire un omomorfismo suriettivo $f: \mathbb{Z} \to G, f(1) = g$ con $g$ generatore di $G$. Le altre associazioni seguono essendo $f$ omomorfismo, infatti $f(x) = f(1 + ... + 1) \text{ (x volte)} = g \cdot g \cdot \ldots \cdot g = g^x$.

Per il primo teorema di isomorfismo, $f(\mathbb{Z}) = G \cong \mathbb{Z}/H$, con $H$ sottogruppo di $\mathbb{Z}$. Siccome $G$ è finito, gli unici sottogruppi ammessi sono i sottogruppi finiti di $\mathbb{Z}$, cioè $\mathbb{Z}/n\mathbb{Z}$. Quindi $G \cong \mathbb{Z}/n\mathbb{Z}$ con $n = |G|$.

Tornando all'esercizio, si ha che $U_9 \cong {\mathbb{Z}}\_6$ e $U_{14} \cong \mathbb{Z}\_6$. Siccome l'isomorfismo è una relazione di equivalenza sull'insieme di tutti i gruppi, si ha che $U_{9} \cong U_{14}$.

---

$U_{24} = \\{ 1, 5, 7, 11, 13, 17, 19, 23 \\}$
$U_{30} = \\{ 1, 7, 11, 13, 17, 19, 23, 29 \\}$

Questi due gruppi non sono ciclici, calcoliamo l'ordine di ogni elemento come suggerito:

Per $U_{24}$:

| $x$ | $o(x)$ |
| --- | ------ |
| 1 | 1 |
| 5 | 2 |
| 7 | 2 |
| 11 | 2 |
| 13 | 2 |
| 17 | 2 |
| 19 | 2 |
| 23 | 2 |

Tutti gli elementi "propri" di $U_{24}$ hanno ordine 2.

Proviamo a calcolare l'ordine degli elementi di $U_{30}$.

| $x$ | $o(x)$ |
| --- | ------ |
| 1 | 1 |
| 7 | 4 |

Possiamo fermarci qui, i due gruppi non possono essere isomorfi, in quanto in $U_{24}$ non c'è nessun elemento di ordine 4.
